src/elements.js

```diff
@@ -220,6 +220,39 @@ _.register({
 		subtree: true
 	},
 
+	"select[multiple]": {
+		extend: "select",
+		selector: "select[multiple]",
+		getValue: element => {
+			const selected = element.selectedOptions;
+			const value = [];
+
+			if (selected.length) {
+				for (const option of selected) {
+					value.push(option.value);
+				}
+			}
+
+			return value;
+		},
+		setValue: (element, value) => {
+			value = Array.isArray(value)? value : [value];
```

Comment:
I'm not sure about expecting an array as a value on a primitive. In most cases, it will not be an array. I would suggest expecting a comma-separated string (convert to array via `.split(/\s*,/)`), but also handling arrays, for the best of both worlds.

Reply:
Fixed that. I even caught a bug, thanks to your comment. Thank you.